src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -481,7 +481,7 @@ public bool Add(T item)
         /// a value that has more complete data than the value you currently have, although their
         /// comparer functions indicate they are equal.
         /// </remarks>
-        public bool TryGetValue(T equalValue, out T actualValue) // TODO-NULLABLE-GENERIC
+        public bool TryGetValue(T equalValue, [MaybeNullWhen(false)] out T actualValue)
```

Comment:
Github won't let me comment there, but while you're here, I noticed that on constructor is missing a question mark for the comparer, we accept null comparers: https://github.com/dotnet/corefx/pull/38309/files#diff-a80c75678ed2d070e638bcb93ba26bccR202

Reply:
Good catch. Done.